spec/src/modules/assistant.js

```diff
@@ -0,0 +1,265 @@
+/* eslint-disable no-unused-expressions, import/no-unresolved */
+import ConstructorIO from '../../../test/constructorio'; // eslint-disable-line import/extensions
```

Comment:
Why do we need to do this differently in this file? It seems to be working fine in other files? I tried changing this to `require` and it seemed to be working but I might be missing something